OneCommunityGlobal/HighestGoodNetworkApp
# Shuhua - Inventory types page

# Description
Implement 5.1.1 List view of ALL inventory types

## Related PRS (if any):
N/A

## Main changes explained:
- Implemented a timestamp to reflect the current time.
- Implemented accordions and tables to render all inventory types by category
- Implemented a table for Units of Measurements

## How to test:
1. check into the current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as any user
5. go to [http://localhost:3000/bmdashboard/inventorytypes](http://localhost:3000/bmdashboard/inventorytypes)
6. Verify the following features:
    1. The timestamp component displays the current time when the page is loaded and can be edited
    2. Blue bars are expandable/collapsible, and the arrows change accordingly
    3. Check the data stored in the `buildingInventoryTypes` collection to make sure all types are listed by category

## Screenshots or videos of changes:
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/83729360/e62878ed-4441-4299-9ad5-cf08d5608eee


## Note:
- The page differs from the original [wireframe](https://www.figma.com/file/kpwXZP2pLtOJ1Fn2u7Sivn/HGN-PII?type=design&node-id=450-6151&mode=design). Changes were made with Jae’s approval to adapt the current data scheme and improve user experience
- The add, edit, and delete features are not implemented yet
- The same for the “Back to the previous page” button
- These errors are unrelated
<img width="1310" alt="potential errors" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/83729360/2d6420e4-7c2a-41be-bd59-8c8951b2ec9a">
